# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
#feedback
bool feed_mode
al_msgs/JointCmd[] feed_joint_pos
geometry_msgs/Pose feed_cartesian_pos

